modules/oxide/src/less/theme/components/collection/collection.less

```diff
@@ -17,18 +17,21 @@
 @collection-item-label-disabled-text-color: fade(@collection-item-label-text-color, 50%);
 @collection-item-caret-disabled-text-color: fade(@collection-item-label-text-color, 50%);
 
-@collection-list-item-active-background-color: @toolbar-button-hover-background-color;
-@collection-list-item-label-active-text-color: contrast(@collection-list-item-active-background-color, @color-black, @color-white);
+@collection-item-hover-background-color: #285ec7;
```

Comment:
This variable needs to be inherited from something else. It shouldn't be hard coded here in the component but be assigned to some other variable so that a skin author just needs to alter the background color at one place.